lib/find/find.js

```diff
@@ -230,7 +230,10 @@ function urlToPath(value, config) {
   numberSignIndex = value.indexOf(numberSign)
   questionMarkIndex = value.indexOf(questionMark)
 
-  if (
+  // Ignore "headings" in image links: `image.png#bordered`
+  if (numberSignIndex !== -1 && nodeType === 'image') {
+    value = value.slice(0, numberSignIndex)
+  } else if (
```

Comment:
I think this is interfering with query params, as can be seen by the fixture change.
This is also not handling the case for full URLs: the `if (url)` before this.

Can this be reverted and added somewhere else?

Reply:
L255 can be solved with:

```js
    return normalize(path.resolve(config.root, value + (nodeType === 'image' ? '' : url.hash)), config)
```

Reply:
Then after the old L239, now L242, add a new:

```js
numberSignIndex = value.indexOf(numberSign)
```

Reply:
The you can add your if-statement on old L241, new L244:

```js
if (numberSignIndex !== -1 && nodeType === 'image') {
  value = value.slice(0, numberSignIndex)
} 
```

Reply:
> L255 can be solved with:
> 
> ```js
>     return normalize(path.resolve(config.root, value + (nodeType === 'image' ? '' : url.hash)), config)
> ```

As I suppose - you mean line 225 :)

It makes code complexity at 21, but the limit is 20 - can you help me with refactor?